src/components/application_manager/test/commands/hmi/get_urls_test.cc

```diff
@@ -234,11 +243,104 @@ TEST_F(GetUrlsTest, ProcessPolicyServiceURLs_SUCCESS) {
             (*command_msg)[am::strings::msg_params][am::hmi_response::urls][0]
                           [strings::url].asString());
 }
+TEST_F(GetUrlsTest, ProcessPolicyServiceURLs_IncorrectIdForSending_UNSUCCESS) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  RequestFromHMIPtr command =
+      PrepareRunFlowAndCreateCommandRequestFromHMI(command_msg);
+
+  EndpointUrls endpoints_;
+  EndpointData data(kDefaultUrl);
+  endpoints_.push_back(data);
+
+  ON_CALL(policy_handler_, GetServiceUrls(kPolicyService, _))
+      .WillByDefault(SetArgReferee<1>(endpoints_));
+  EXPECT_CALL(policy_handler_, GetServiceUrls(kPolicyService, _));
```

Comment:
@dcherniev, what does this should mean? First you use `ON_CALL` for `GetServiceUrls()` and on the next line you use `EXPECT_CALL` for the same function